docs/deployment/universal-profile.md

```diff
@@ -159,31 +200,242 @@ await lspFactory.LSP3UniversalProfile.deploy({
 });
 ```
 
-## Uploading LSP3 metadata to IPFS
+### Uploading LSP3 metadata to IPFS
 
-If you wish to upload your LSP3 metadata before deploying you can do so using the static `uploadProfileData` method. This uses the same `lsp3Profile` object schema defined above when deploying a Universal Profile:
+You can upload your LSP3 metadata before deploying a Universal Profile by using the `uploadProfileData` method. This uses the same `lsp3Profile` object schema defined [above](./universal-profile#adding-lsp3-metadata) when deploying a Universal Profile.
```

Comment:
It might be more visible to put the anchor link on "[`lsp3Profile` oject schema]", instead of on "[above]"